exercises/simple-linked-list/simple_linked_list.py

```diff
@@ -0,0 +1,27 @@
+class Node(object):
+    def value(self):
+        pass
+
+    def next(self):
+        pass
+
+
+class LinkedList(object):
+    def __len__(self):
```

Comment:
I think we should include `__init__` here (and above) - they are necessary methods and are expected to take an argument.

Reply:
Good catch; I meant to add that.